### PR TITLE
Added 3rd Party Fieldtype

### PR DESCRIPTION
**Pull request by serkanalgur**
Just created a new 3rd party fieldtype for selecting Font Awesome Icons.
[CMB2 Field Type: Font Awesome Icon Selector](https://github.com/serkanalgur/cmb2-field-faiconselect)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ A complete list of all our awesome contributors found here: [github.com/CMB2/CMB
 * [CMB2 Field Type: Ajax Search](https://github.com/rubengc/cmb2-field-ajax-search) by [rubengc](https://github.com/rubengc): Based on [CMB2 Field Type: Post Search Ajax](https://github.com/alexis-magina/cmb2-field-post-search-ajax), adds the ability to attach posts/users/terms, and the ability to limit the maximum number of attached objects.
 * [CMB2 Field Type: Visual Style Editor](https://github.com/rubengc/cmb2-field-visual-style-editor) by [rubengc](https://github.com/rubengc): Custom field for CMB2 which allows customizing style from a small set of controls.
 * [CMB2 Field Type: CMB2 Roadway Segments](https://github.com/pixelwatt/cmb2-roadway-segments) by [pixelwatt](https://github.com/pixelwatt): This plugin adds a new CMB2 fieldtype for drawing roadway segments onto a map and provides a shortcode for display.
+* [CMB2 Field Type: Font Awesome Icon Selector](https://github.com/serkanalgur/cmb2-field-faiconselect) by [serkanalgur](https://github.com/serkanalgur): This plugin add a new CMB2 fieldtype for selecting font awesome icons.
```
Comment (jtsternberg) [inline README.md]: Please change to “this plugin adds” and “field type” and title case “Font Awesome”. Thank you!

Comment (serkanalgur) [reply README.md]: Changed :+1:

**Comment by jtsternberg** [comment]
That works, thanks.

**Comment by serkanalgur** [comment]
You are welcome :+1:

**Pull request merged**